packages/checkout/widgets-lib/src/resources/text/textConfig.ts

```diff
@@ -364,6 +364,10 @@ export const text = {
           primaryAction: 'Try again',
           secondaryAction: 'Dismiss',
         },
+        [SaleErrorTypes.INVALID_PARAMETERS]: {
+          description: 'Sorry, we\'re unable to process your order. Missing information from client',
+          secondaryAction: 'Dismiss',
+        },
       },
```

Comment:
@dreamoftrees do we still need to update this file?

Reply:
Good catch @andrearampin @jhesgodi the textConfig should be depreciated and I should remove in a fresh PR. Any text changes should be included in the `locales/en.json` and supported by the i18next hooks like `t()`

Reply:
thank you both. fixed